tests/build-hook.nix

```diff
@@ -1,15 +1,22 @@
-{ busybox }:
+{ busybox, contentAddressed ? false }:
 
 with import ./config.nix;
```

Comment:
I recall you had rigged this up so that we got a CA or no CA by default `mkDerivation` by default?